File: 00-Code/JavaEE/SpringInAction/Chapter03/src/test/java/me/ztiany/bean/runtimedi/Person.java
package me.ztiany.bean.runtimedi;

/**
 * @author Ztiany
 * Email ztiany3@gmail.com
 * Date 18.6.15 0:06
 */
public class Person {

    private int age;
    private String name;

    public Person(int age, String name) {
        this.age = age;
        this.name = name;
    }

    @Override
    public String toString() {
        return "Person{" +
                "age=" + age +
                ", name='" + name + '\'' +
                '}';
    }
}

File: 00-Code/Android/JsonAnalyze/app/src/main/java/me/ztiany/jsonanalyze/kotlin/Person.kt
package me.ztiany.jsonanalyze.kotlin

/**
 *
 *@author Ztiany
 *      Date : 2018-08-13 13:36
 */
const val PERSON_JSON = """{"name":"Ztiany","age":28,"address":"SZ"}"""

data class Person(val name: String, val age: Int, val address: String)